tests/unit/ec2/autoscale/test_group.py

```diff
@@ -603,6 +603,54 @@ def check_tag_attributes_set(self, name, value, attr):
         else:
             self.assertEqual(getattr(tag, attr), value)
 
+class TestAutoScalingTagFilter(AWSMockServiceTestCase):
+    connection_class = AutoScaleConnection
+
+    def default_body(self):
+        return b"""
+        <DescribeTagsResponse>
+            <ResponseMetadata>
+                <RequestId>requestId</RequestId>
+            </ResponseMetadata>
+        </DescribeTagsResponse>
+        """
+
+    def test_get_all_tags(self):
+        self.set_http_response(status_code=200)
+
+        tags = [
```

Comment:
Ah, you're right!  Call it a consequence of too much copy-paste as I learned how to write a boto test.  Cheers.